src/setupTest.js

```diff
@@ -1 +1,69 @@
-import 'babel-polyfill';
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
+import '@testing-library/jest-dom';
+import React from 'react';
+import { render as rtlRender } from '@testing-library/react';
+import AppProvider from '@edx/frontend-platform/react/AppProvider';
+import { configure as configureI18n, IntlProvider } from '@edx/frontend-platform/i18n';
+import { configure as configureLogging, MockLoggingService } from '@edx/frontend-platform/logging';
+import { getConfig, mergeConfig } from '@edx/frontend-platform';
+import { configure as configureAuth, MockAuthService } from '@edx/frontend-platform/auth';
+import appMessages from './i18n';
+
+Object.defineProperty(window, 'matchMedia', {
+  writable: true,
+  value: jest.fn().mockImplementation((query) => ({
+    matches: false,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(), // Deprecated
+    removeListener: jest.fn(), // Deprecated
+    addEventListener: jest.fn(),
+    removeEventListener: jest.fn(),
+    dispatchEvent: jest.fn(),
+  })),
+});
+
+export function initializeMockApp() {
+  mergeConfig({
+    // MICROBA-1505: Remove this when we remove the flag from config
```

Comment:
Is this comment still correct? Do we still need this `mergeConfig` section? Is this some accidental copy-paste? I checked MICROBA-1505 and this has something to do with the Communications MFE, and isn't related to the Learner Record MFE.

Reply:
Definitely copy-pasta from the Comms MFE so I'll get rid of that as well

Reply:
Thank you!